assets/queries/terraform/aws/cloudtrail_log_files_s3_bucket_is_publicly_accessible/query.rego

```diff
@@ -20,3 +20,18 @@ CxPolicy[result] {
 		"keyActualValue": sprintf("aws_s3_bucket[%s] is publicly accessible", [s3BucketName]),
 	}
 }
+
+CxPolicy[result] {
+	module := input.document[i].module[name]
+	keyToCheck := common_lib.get_module_equivalent_key(module.source, "aws_s3_bucket", "acl")
+	publicAcl := {"public-read", "public-read-write"}
```

Comment:
`publicAcl := {"public-read", "public-read-write"}` could be defined outside of both rules (to avoid double declaration)